SU-HSDO/suhumsci
# HSD8-355 Change config ignore to only ignore some parts of google ana…

# READY FOR REVIEW

# Summary
- Allow for some google analytics config to be synced and some to be ignored
- This allows for the config to be installed when a new site is created and prevents a php loading error.

# Needed By (Date)
- End of sprint. 2/13

# Urgency
- high

# Steps to Test
1. Checkout this branch
1. composer install
1. edit `vendor/acquia/blt/settings/config.settings.php` file add `$split = 'prod';` around line 87
1. install a new site `drush @sitename.local si config_installer -y`
1. import config `drush @sitename.local cim -y`
1. log into the site `drush @sitename.local uli --uri=sitename.suhumsci.loc`
1. visit `/admin/config/system/google-analytics` page and ensure no error display

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)…ltyics
